ci/pipeline.yml

```diff
@@ -36,20 +36,41 @@ resources:
     regexp: releases/v(.*)/cf-cli_(.*)_linux_x86-64.tgz
     region_name: us-west-1
 
+- name: v6-cli-binary-windows-64
+  type: s3
+  source:
+    bucket: cf-cli-releases
+    regexp: releases/v(.*)/cf-cli_(.*)_winx64.tgz
```

Comment:
Just checked the s3 buckets and windows binaries are .zip files

Reply:
Thanks for catching this @jdgonzaleza.